web/packages/teleterm/src/services/pty/ptyHost/buildPtyOptions.ts

```diff
@@ -46,9 +52,28 @@ export async function buildPtyOptions(
   cmd: PtyCommand
 ): Promise<{
   processOptions: PtyProcessOptions;
+  openedShell: Shell;
```

Comment:
What does the `opened` prefix signifies, why not call this field `shell` instead?

Reply:
I wanted to show that the `openedShell` can be different from the `shellId` that we sent to `buildPtyOptions`, but this name is misleading as per your other comment. Removed.